ingestion/README.md

```diff
@@ -140,11 +140,12 @@ To upload dataset files such as 3D neuron models, EM tile images and segmentatio
 When using the `add-dataset` subcommand, don't forget to specify the dataset ID corresponding to the files you're uploading (we will remember you otherwise).
 The following flags help determine which files to upload:
 
-- `-s`/`--segmentation`: Path to the directory or files containing segmentation data.
+- `-seg`/`--segmentation`: Path to the directory or files containing segmentation data.
```

Comment:
I would just modify `files containing segmentation data` to `files containing neuron segmentation data` as the data for the synapses are segmentation data also.